platform_API.md

```diff
@@ -50,6 +50,19 @@ Authorization: Key <your Server API Key>
 
 ## API Reference
 
+### Authentication
+
+#### Access a user's resource
```

Comment:
Maybe add a colon here? "Access a user's resource:"

Reply:
Thanks for catching this!

Reply:
Payment in acaunt

Reply:
cbttdpfyhsj2guu6rcnzwctfijniocixzovlvjlxrvadppnedrxb1pnymcikqooy